src/ui/components/TestSuite/views/GroupedTestCases/Panel.tsx

```diff
@@ -71,6 +71,7 @@ export default function Panel() {
             className={styles.Attribute}
             icon="schedule"
             label={getTruncatedRelativeDate(recording.date)}
+            title={recording.date}
```

Comment:
This format isn't the easiest to read. Can we show a _formatted_ full date?

Reply:
yeah, how would you prefer it to be formatted?

Reply:
Maybe?
```js
`${date.toLocaleDateString()} ${date.toLocaleTimeString()}`
```

> 6/23/2023 5:39:32 PM

Reply:
Yep, that works nicely:

![image](https://github.com/replayio/devtools/assets/1128784/6f6d779a-3b0a-441d-abce-aa8f994be063)

Reply:
Love it.